### ISSUE TITLE
[Web] Support Elasticsearch 8.X

### ISSUE THREAD

**Issue by nicpenning** [cert-ee/cuckoo3#178]
The current supported version of Elasticsearch for Cuckoo is 7.X but it would be great to use the latest version. [EOL dates](https://www.elastic.co/support/eol). It seems Elastic is working on 9.0 so 7.X could be unsupported upon its arrival which is still unknown. Best to support Elasticsearch latest 8.X version if Elasticsearch is still planning on being used which I think would be great. 

Also, in coming versions, Elasticsearch will be open source again so that is great as well [article](https://www.elastic.co/blog/elasticsearch-is-open-source-again).

https://github.com/cert-ee/cuckoo3/blob/64d9aabd6abe4f037968b81587e942621ab30e12/common/setup.py#L44-L45

**Comment by cert-ee-raidar** [comment]
@nicpenning 
Yes, we are currently updating all Cuckoo3 Python dependencies to their latest versions.
We should have it ready soon.
